spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'test', posts_counter: 0) }
+
+  before { subject.save }
+
+  it 'should be valid' do
+    expect(subject).to be_valid
+  end
+
+  it 'should be valid' do
```

Comment:
## [OPTIONAL] 

- [ ] Kindly change this message to `should not be valid` as is testing the invalidity of the user.

Reply:
Thanks!